test.py

```diff
@@ -0,0 +1,3 @@
+from dateparser import parse
+
+print(type(None))
```

Comment:
Is this file intentional?

Reply:
This file is temporary only for local tests or in meeting in shall remove that now thanks

Reply:
I have removed this is now. 👍